ansible_base/authentication/models/authenticator.py

```diff
@@ -52,6 +52,31 @@ class Authenticator(UniqueNamedCommonModel):
         on_delete=SET_NULL,
     )
 
+    def save_default_jwt_algorithms(self):
```

Comment:
We should move this out of the model. It shouldn't know about specifics for requirements of any of the plugins.